tools/itch5bookdepth.cpp

```diff
@@ -42,10 +42,11 @@ class config : public jb::config_object {
 };
 
 /// Record the book depth
+template <typename book_type>
```

Comment:
As a side note, this means (I think) that you could reuse this file when you implement your own cache_aware_book.  That would be super neat.